AdminServer/appscale/admin/__init__.py

```diff
@@ -886,8 +893,16 @@ def version_from_payload(self):
       raise CustomHTTPError(HTTPCodes.BAD_REQUEST, message=message)
 
     desired_fields = update_mask.split(',')
-    supported_fields = {'appscaleExtensions.httpPort',
-                        'appscaleExtensions.httpsPort'}
+    supported_fields = {
+      'appscaleExtensions.httpPort',
+      'appscaleExtensions.httpsPort',
+      'automaticScaling.standard_scheduler_settings.max_instances',
```

Comment:
Should this be `standardSchedulerSettings`?

Reply:
My understanding is that these `automaticScaling` names should be as defined in the api doc here:

https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions/patch

Reply:
I think what's happening is that Google's server does some automatic case conversion that applies to _all_ keys. After testing, the API accepts both `automaticScaling.standardSchedulerSettings.maxInstances` and `automatic_scaling.standard_scheduler_settings.max_instances` in the payload of the patch.

It's annoying that their documentation isn't consistent, but I'm fine with what you have here.